utility/types/gov.go

```diff
@@ -18,7 +18,8 @@ const (
 	ServiceNodeMaxPauseBlocksParamName     = "service_node_max_pause_blocks"
 	ServiceNodesPerSessionParamName        = "service_nodes_per_session"
 
-	FishermanMinimumStakeParamName       = "fisherman_minimum_stake"
+	FishermanMinimumStakeParamName = "fisherman_minimum_stake"
+	// DISCUSS: Should this parameter be removed?
```

Comment:
@andrewnguyen22 

The specification (https://github.com/pokt-network/pocket-network-protocol/tree/main/utility) only has relay chain IDs for application and service nodes and doesn't have one for fisherman.

What is the use of `FishermanMaxChainsParamName`?

Reply:
https://github.com/pokt-network/pocket/pull/303#discussion_r998776679

Reply:
https://github.com/pokt-network/pocket/pull/303#discussion_r999653010